### PR TITLE
ENG-14923, better naming for @Statistics EXPORT, add LastQueuedTimest…

### PR DESCRIPTION
**Pull request by luyangco**
…amp and LastAckedTimestamp column.

Change-Id: I6f3f3953b35de1669d696684a320ac90e1b29fbe

### CODE REVIEW

File: src/ee/storage/ExportTupleStream.cpp [hunk 1]
```diff
@@ -172,6 +172,8 @@ size_t ExportTupleStream::appendTuple(int64_t lastCommittedSpHandle,
     const size_t startingUso = m_uso;
     m_uso += (streamHeaderSz + io.position());
     m_exportSequenceNumber++;
+    // TODO: better to only add this before commit
```
Comment (rdykiel) [inline src/ee/storage/ExportTupleStream.cpp]: Why is this still a TODO? Is there something else we should do here?

File: src/frontend/org/voltdb/ExportStatsBase.java [hunk 2]
```diff
@@ -25,34 +25,36 @@
...
+            m_sourceName = sourceName;
             m_exportTarget = exportTarget;
+            m_isExporting = isExporting ? "TRUE" : "FALSE";
             m_tupleCount = tupleCount;
             m_tuplesPending = tuplesPending > 0 ? tuplesPending : 0;
+            m_lastQueuedTimestamp = lastQueuedTimestamp;
+            m_lastAckedTimestamp = lastAckedTimestamp;
             m_averageLatency = averageLatency;
             m_maxLatency = maxLatency;
+            m_missingTuple = missingTuple;
```
Comment (rdykiel) [inline src/frontend/org/voltdb/ExportStatsBase.java]: should it be something like "gapCount" ?

File: tests/frontend/org/voltdb/export/TestExportDataSource.java [hunk 3]
```diff
@@ -214,23 +214,23 @@ public void run() {
             int buffSize = 20 + StreamBlock.HEADER_SIZE;
             ByteBuffer foo = ByteBuffer.allocateDirect(buffSize);
             foo.duplicate().put(new byte[buffSize]);
-            s.pushExportBuffer(1, 1, foo, false);
+            s.pushExportBuffer(1, 1, 0, foo, false);
```
Comment (rdykiel) [inline tests/frontend/org/voltdb/export/TestExportDataSource.java]: You should be consistent: some place you're using "L" and some not.

File: src/ee/storage/ExportTupleStream.cpp (hunk 1 above)
Comment (luyangco) [reply src/ee/storage/ExportTupleStream.cpp]: Removed TODO. When I marked this as TODO I was thinking it's nice to have export stream that doesn't reply on next insert to trigger the commit. But this is beyond the scope of current ticket.

File: src/frontend/org/voltdb/ExportStatsBase.java (hunk 2 above)
Comment (luyangco) [reply src/frontend/org/voltdb/ExportStatsBase.java]: Changed.

File: tests/frontend/org/voltdb/export/TestExportDataSource.java (hunk 3 above)
Comment (luyangco) [reply tests/frontend/org/voltdb/export/TestExportDataSource.java]: Changed.

**Comment by luyangco** [comment]
It turns out the order of first a few columns that the scripts care about is not changed, so I didn't modify those files.

**Pull request merged**